docker-compose.yml

```diff
@@ -397,7 +397,7 @@ services:
       - MYSQL_DATABASE=${_APP_DB_SCHEMA}
       - MYSQL_USER=${_APP_DB_USER}
       - MYSQL_PASSWORD=${_APP_DB_PASS}
-    command: 'mysqld --innodb-flush-method=fsync --wait_timeout=86400' # add ' --query_cache_size=0' for DB tests
+    command: 'mysqld --innodb-flush-method=fsync' # add ' --query_cache_size=0' for DB tests
```

Comment:
This also need to be removed from `app/views/install/compose.phtml`

That file serves as a template for the `docker-compose.yml`  file that will end up with the end-user.

Reply:
end-developer*